src/main/kotlin/com/hypto/iam/server/security/Authentication.kt

```diff
@@ -102,6 +102,13 @@ class TokenAuthenticationProvider internal constructor(
             if (!authSchemeExists) {
                 return@tokenAuthenticationCredentials it
             }
+            if (call.request.headers.contains("x-issuer")) {
+                if (!it.startsWith("Bearer")) {
+                    throw AuthenticationException("Invalid token")
```

Comment:
Why is this case ? Do we expect to always get the token with bearer auth ? Is this done at front-end side ?